### PR TITLE
chore: remove target supported destinations

### PR DESCRIPTION
**Pull request by markusfassbender**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

In Xcode the target supported different destinations, but we only develop the iOS and iPad app.

### Solutions

Remove all unsupported destinations.

#### How to Test

- No need to test

Screenshot 2024-01-10 at 11 15 44

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by netbe** [comment]
it does run on ipad

**Comment by markusfassbender** [comment]
@netbe whoops I looked at the App Store and saw this:

Screenshot 2024-01-10 at 12 29 28

Then I will add iPad again, thanks!

**Review by agisilaos — APPROVED** [review]

File: wire-ios/Wire-iOS.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -10571,6 +10571,10 @@
 				PROVISIONING_PROFILE_SPECIFIER = "$(PROVISIONING_PROFILE_SPECIFIER)";
 				SDKROOT = iphoneos;
 				SKIP_INSTALL = YES;
+				SUPPORTED_PLATFORMS = "iphoneos iphonesimulator";
+				SUPPORTS_MACCATALYST = NO;
+				SUPPORTS_MAC_DESIGNED_FOR_IPHONE_IPAD = NO;
+				SUPPORTS_XR_DESIGNED_FOR_IPHONE_IPAD = NO;
```
Comment (netbe) [inline wire-ios/Wire-iOS.xcodeproj/project.pbxproj:10577]: what's this one?

Comment (markusfassbender) [reply wire-ios/Wire-iOS.xcodeproj/project.pbxproj:10577]: Setting name: SUPPORTS_XR_DESIGNED_FOR_IPHONE_IPAD
Show the Apple Vision (Designed for iPhone) and Apple Vision (Designed for iPad) destinations.
> https://developer.apple.com/documentation/xcode/build-settings-reference

**Review by netbe — APPROVED** [review]

**Pull request merged**